src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -6799,6 +6967,29 @@ private static string GetNumbersExpectedString(bool prettyPrint, int numberOfEle
             return Encoding.UTF8.GetString(ms.ToArray());
         }
 
+        private static string GetNumbersExpectedString<T>(bool prettyPrint, int numberOfElements, T value)
```

Comment:
Could you use this for all the other callers of `GetNumbersExpectedString(bool prettyPrint, int numberOfElements)` and remove that one?
https://github.com/dotnet/corefx/blob/91ffa88d29cac706db3bceac055080c1126c6ea9/src/System.Text.Json/tests/Utf8JsonWriterTests.cs#L6780

Also, as a nit, consider using a `StringBuilder` and `StringWriter` instead of a `MemoryStream`/`StreamWriter` (that forces you to transcode from UTF-8 to string again). I understand that existing tests use `MemoryStream` in a bunch of places, but since you are updating this one, consider using the simpler approach for the **new** code.

Reply:
I have done similar changes to use `GetPropertyExpectedString` in place of `GetIntegerExpectedString`. Please check.